modules/statistics/statistics.js

```diff
@@ -72,7 +72,8 @@ function _initCallStatsBackend(options) {
         applicationName: options.applicationName,
         getWiFiStatsMethod: options.getWiFiStatsMethod,
         confID: options.confID,
-        siteID: options.siteID
+        siteID: options.siteID,
+        configParams: options.configParams
```

Comment:
document configParams in the Statistics function.

Reply:
commented